framework/src/media/audio/audio_manager.h

```diff
@@ -106,12 +113,12 @@ audio_manager_result_t set_audio_stream_in(unsigned int channels, unsigned int s
  *   output card. If the target sample rate is out of range from the sample rates
  *   supported by the active output audio card, a resampling flag is set.
  *
- * Input parameters:
+ * Input parameter(s):
```

Comment:
This is not open API, instead of (s), please update properly.

Reply:
Modified.